src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -42,6 +42,32 @@ define(function (require, exports, module) {
     var STEP_MULTIPLIER = 5;
 
     /**
+     * Convert 0x notation into hex6 format for tinycolor
+     * compatibility: ("0xFFAACC" => "#FFFFFF")
+     */
+    function color0xToHex(color,convertToStr){
+        var format0xToHexColor = color.replace("0x","#");
+        var hexColor = tinycolor(format0xToHexColor);
+        hexColor._format = "0x";
+
+        if(convertToStr){
```

Comment:
One more nit: `if (convertToString) {` (we really should enable http://eslint.org/docs/rules/space-after-keywords and similar in our config, our spaces are wildly inconsistent across the codebase 🤔 )